src/html.js

```diff
@@ -11,9 +11,37 @@ export default function HTML(props) {
                     name="viewport"
                     content="width=device-width, initial-scale=1, shrink-to-fit=no"
                 />
+                <script
```

Comment:
shouldnt we have <script async="" src="//www.googletagmanager.com/gtm.js?id=GTM-XXX"></script> ?

Reply:
Yes, thanks!